identity-service/src/notifications/sendNotifications/publishNotifications.js

```diff
@@ -106,7 +114,7 @@ const publishNotifications = async (notifications, metadata, userNotificationSet
     const userId = getPublishUserId(notification, publishNotifType)
     const types = getPublishTypes(userId, publishNotifType, userNotificationSettings)
 
-    if (solanaNotificationBaseTypes.includes(publishNotifType)) {
+    if (solanaNotificationBaseTypes.includes(notification.type)) {
```

Comment:
This seems weird. Do you want to only listen for specific Milestone types?

Reply:
I only wanted to include it if it's for milestone listen, not the base notification which is `milestone` because those are handled differently